### PR TITLE
Re-Enable Rat King

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
Title

## Why / Balance
Very high player demand for this antag and more antags in general. 

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: The King of Rats has returned to Delta Sector stations

### CODE REVIEW

**Comment by Radezolid** [comment]
Bad idea, it will happen the same things as before.

File: Resources/Prototypes/DeltaV/GameRules/events.yml [hunk 1]
```diff
@@ -132,13 +132,13 @@
...
-#  id: RatKingSpawn
-#  components:
-#  if you uncomment this update this for antag refactor, below wont work
-#  - type: MidRoundAntagRule
-#    spawner: SpawnPointGhostRatKing
+- type: entity
+  parent: BaseMidRoundAntag
+  id: RatKingSpawn
+  components:
+  if you uncomment this update this for antag refactor, below wont work
```
Comment (Radezolid) [inline Resources/Prototypes/DeltaV/GameRules/events.yml:139]: This shouldn't be here it was a whole comment so either comment it back or delete it

**Comment by MilonPL** [comment]
After getting feedback from direction, I will be closing this PR. I'm sure someone will elaborate on it further, but the general consensus seems to be "we do not want rat kings in their current implementation".

**Comment by Lyndomen** [comment]
Howdy, yes to confirm, we are not a fan of this antagonist with its current implementation. To talk about our future plans, we 100% hear you regarding wanting more antagonist variety, and I'm spending some time with the team to organize these efforts.

**Pull request closed without merging**